src/app/clients/clients-view/general-tab/general-tab.component.html

```diff
@@ -555,7 +555,7 @@ <h3>Shares Accounts</h3>
 
     <tr mat-header-row *matHeaderRowDef="openSharesColumns"></tr>
     <tr mat-row *matRowDef="let row; columns: openSharesColumns;"
-      [routerLink]="['../', 'sharesaccounts', row.id, 'transactions']"></tr>
```

Comment:
why this?

Reply:
won't you navigate for the closed recurring deposits account?

Reply:
won't you navigate for the closed recurring deposits account?

Reply:
Its the shares account table